src/Workspaces/Core/MSBuild.BuildHost/Build/ProjectBuildManager.cs

```diff
@@ -66,7 +66,7 @@ internal class ProjectBuildManager
         {
```

Comment:
> If we don't, the finalizer can run and tear down the test process when
other tests are still running.

I odn't have a problem with this change, but the explaantion doesn't really make sense ot me.  being in a finally block shouldn't change anything *except* for having that code run in the case of early-exit/exception.  Which shoudln't apply in the code below because nothing seems exceptional.

Reply:
So when I was working on the change, I did break things at times where exceptions were being thrown. Since the process would then tear down I woudln't even see the full test results!

In a later commit these tests got rewritten anyways.